app/workers/send_email_worker.rb

```diff
@@ -34,7 +34,7 @@ def self.perform_async_in_queue(email_id, metrics = {}, queue:)
 private
 
   def rate_limit_exceeded?
-    rate_limiter.exceeded?("delivery_request",
+    rate_limiter.exceeded?("requests",
```

Comment:
This change will need a bit of careful handling: we'll temporarily reset our counter and stop rate limiting.

Reply:
Yup, we already have to be careful because of the change of queue names too